RevenueCatUI/UIKit/PaywallViewController.swift

```diff
@@ -141,7 +145,11 @@ public protocol PaywallViewControllerDelegate: AnyObject {
     @objc(paywallViewControlle:didChangeSizeTo:)
     optional func paywallViewController(_ controller: PaywallViewController,
                                         didChangeSizeTo size: CGSize)
-
+    
+    /// Notifies that a purchase has been initiated in a ``PaywallViewController``.
+    @objc(paywallViewController:didSelectPackage:)
+    optional func paywallViewController(_ controller: PaywallViewController,
+                                        didSelectPackage package: Package)
```

Comment:
the naming is somewhat confusing. 
I'd guess this callback is triggered when I switch from Yearly to Monthly package.
`PurchaseStarted` would be better.

Reply:
Right! I updated the method to `paywallViewController(_:didInitiatePurchaseWithSelectedPackage:)` to better reflect the reported action.